src/test/ui-fulldeps/session-diagnostic/diagnostic-derive.stderr

```diff
@@ -392,6 +392,7 @@ LL | #[derive(SessionDiagnostic)]
              Ident
              String
              Symbol
+             rustc_hir::hir::ConstContext
```

Comment:
I think I might want to normalize this part of the test output because it's prone to changing like this whenever a `IntoDiagnosticArg` impl is added.

Reply:
Added a commit to #98624 to do this.